sdk/containerregistry/azure-containerregistry/tests/testcase.py

```diff
@@ -202,47 +163,29 @@ def get_credential(self):
     def create_registry_client(self, endpoint, **kwargs):
         return ContainerRegistryClient(endpoint=endpoint, credential=self.get_credential(), **kwargs)
 
-    def create_container_repository(self, endpoint, name, **kwargs):
-        return ContainerRepository(endpoint=endpoint, name=name, credential=self.get_credential(), **kwargs)
-
     def create_anon_client(self, endpoint, **kwargs):
         return ContainerRegistryClient(endpoint=endpoint, credential=None, **kwargs)
 
-    def assert_content_permission(self, content_perm, content_perm2):
-        assert isinstance(content_perm, ContentProperties)
-        assert isinstance(content_perm2, ContentProperties)
-        assert content_perm.can_delete == content_perm2.can_delete
-        assert content_perm.can_list == content_perm2.can_list
-        assert content_perm.can_read == content_perm2.can_read
-        assert content_perm.can_write == content_perm2.can_write
-
-    def assert_tag(
-        self,
-        tag,
-        created_on=None,
-        digest=None,
-        last_updated_on=None,
-        content_permission=None,
-        name=None,
-        registry=None,
-        repository=None,
-    ):
-        assert isinstance(tag, ArtifactTagProperties)
-        assert isinstance(tag.writeable_permissions, ContentProperties)
-        assert isinstance(tag.created_on, datetime)
-        assert isinstance(tag.last_updated_on, datetime)
-        if content_permission:
-            self.assert_content_permission(tag.content_permission, content_permission)
-        if created_on:
-            assert tag.created_on == created_on
-        if last_updated_on:
-            assert tag.last_updated_on == last_updated_on
-        if name:
-            assert tag.name == name
-        if registry:
-            assert tag.registry == registry
-        if repository:
-            assert tag.repository == repository
+    def set_all_properties(self, properties, value):
+        properties.can_delete = value
+        properties.can_read = value
+        properties.can_write = value
+        properties.can_list = value
+        # try:
+        #     properties.teleport_enabled = value
+        # except AttributeError:
+        #     pass
+        return properties
+
+    def assert_all_properties(self, properties, value):
+        assert properties.can_delete == value
+        assert properties.can_read == value
+        assert properties.can_write == value
+        assert properties.can_list == value
+        # try:
```

Comment:
nit: can you either uncomment or remove the commented out code? unless you're planning on uncommenting later